go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/fortytw2/leaktest v1.3.0 // indirect
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/go-gorp/gorp v2.0.0+incompatible // indirect
-	github.com/go-ldap/ldap v3.0.3+incompatible // indirect
+	github.com/go-ldap/ldap v3.0.3+incompatible
```

Comment:
We are no longer using go-ldap directly because there was a bug with "tracing" so we need to fork it to fix the problem (The PR is waiting to get merged since some months ago already).

Reply:
Fixed :+1: